packages/components/app/styles/components/gutters.scss

```diff
@@ -0,0 +1,51 @@
+//
+// GUTTERS
+//
+// properties within each class are sorted alphabetically
+//
+
+// Loose spacing scale, move out to centralize for re-use:
+// (See: https://www.figma.com/file/oQsMzMMnynfPWpMEt91OpH/HDS-Product---Foundations?node-id=2008%3A3)
+:root {
+  --hds-spacing-xxs: 4px;
+  --hds-spacing-xs: 8px;
+  --hds-spacing-sm: 12px;
+  --hds-spacing-md: 16px;
+  --hds-spacing-lg: 24px;
+  --hds-spacing-xl: 36px;
+  --hds-spacing-xxl: 48px;
+}
+
+// TODO: Determine best default spacing size
+$default-spacing: var(--hds-spacing-md); // set as Sass variable to not expose
+
+.hds-gutters {
+  // vertical (make default ?): (Find out most common use cases)
+  // &:not([class*="hds-gutters-horizontal"]), // yes/ no?
+  &-vertical {
+    > * {
+      &:not(li) {display: block;}
```

Comment:
I wanted to avoid using flex box for this simple component (at least for first iteration) but I see that changing the display of the children like this has potential issues so I'm going to try using flexbox after all...

Btw, after I get this “Gutters” component into better shape and have an API others agree with we still don't have to necessarily merge it right away (or at all). I could work on the more complex “Layout” component I have planned next using what we've agreed upon from this one. (I wanted to start with this simpler component first to use as a basis for the more complex one.)